tests/dags/common/loader/test_reporting.py

```diff
@@ -0,0 +1,26 @@
+from unittest import mock
+
+import pytest
+from common.loader.reporting import report_completion
+
+
+@pytest.fixture(autouse=True)
+def send_message_mock() -> mock.MagicMock:
+    with mock.patch("common.slack.send_message") as SendMessageMock:
+        yield SendMessageMock.return_value
+
+
+@pytest.mark.parametrize(
+    "should_send_message",
+    [
+        (True,),
+        (False,),
+    ],
+)
+def test_report_completion(should_send_message, send_message_mock):
```

Comment:
If `autouse` is set to `True` on the fixture above, we don't need to explicitly add `send_message_mock` in the test parameters here.